rs/nns/governance/README.md

```diff
@@ -0,0 +1,9 @@
+The (NNS) governance canister is the "heart" of the Network Nervous System (NNS). This is where
```

Comment:
NCR;

I thing the NNS Gov is more similar to the brain of the NNS. 

I'd suggest reserving the heart analogy for the CMC.

Wdyt?

Reply:
I think "heart" usually means the central, most important component. E.g. see definition [6b here](https://www.merriam-webster.com/dictionary/heart).

I could just skip this figurative language.